### PR TITLE
lp1719474 Follow-up: Fix navigation usability issues in sidebar tree

### PR DESCRIPTION
**Pull request by uklotzde**
From the discussion in #1619

A timeout of ~~250~~300 ms still feels responsive and scrolling slowly through the list of playlists and crates is possible. This new timeout is now used for every LibraryFeature ~~as a default, unless customized in a derived class~~!

### CODE REVIEW

**Comment by uklotzde** [comment]
- Cleaner and simpler code
- A single constant for consistent behavior within all feature sections
- No more collateral damage = additional latency when using the mouse

I've increased the timeout to 300 ms which works very well for me in most situations.

**Comment by uklotzde** [comment]
Did a forced update to avoid changes in WLibrarySidebar that were unnecessary.

**Comment by daschuer** [comment]
Works good and LGTM.

File: src/library/library.cpp [hunk 1]
```diff
@@ -210,6 +210,8 @@ void Library::bindSidebarWidget(WLibrarySidebar* pSidebarWidget) {
     connect(m_pSidebarModel, SIGNAL(selectIndex(const QModelIndex&)),
             pSidebarWidget, SLOT(selectIndex(const QModelIndex&)));
     connect(pSidebarWidget, SIGNAL(pressed(const QModelIndex&)),
+            m_pSidebarModel, SLOT(pressed(const QModelIndex&)));
+    connect(pSidebarWidget, SIGNAL(clicked(const QModelIndex&)),
```
Comment (daschuer) [inline src/library/library.cpp]: This change causes that the activate() is called twice.

**Pull request merged**